app/assets/stylesheets/templates/_admin.scss

```diff
@@ -1,16 +1,14 @@
 .admin {
   background: white;
-
   .hub-content-wrapper {
     margin-top: 3rem;
   }
-
   .hub-section {
     margin: 3.5rem;
   }
-
   .page-wrapper {
     padding: 0;
+    overflow: unset;
```

Comment:
maybe make a note of why this is here or use mixin to set this so we can find to remove it later.